package.json

```diff
@@ -55,7 +55,7 @@
     "postman": "npm run bundle && openapi2postmanv2 --pretty --spec dist/$npm_package_config_bundlefile --output dist/$npm_package_config_postmanfile",
     "pretty": "prettier --write .",
     "pretty:check": "prettier --check .",
-    "proxy": "prism proxy --errors $npm_package_config_specfile https://api.lob.com/v1",
+    "proxy": "prism proxy $npm_package_config_specfile https://api.lob.com/v1",
```

Comment:
Is the `errors` flag unnecessary now?

Reply:
Unfortunately, the `errors` flag messes up the response. I'm actually going to convert this to a draft and wait to consult with Hilary, because the fix which was previously implemented doesn't seem to cover this.